src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -472,5 +1117,158 @@ internal static unsafe float Dist2(Span<float> src, Span<float> dst)
             }
         }
 
+        internal static unsafe void ZeroItemsU(AlignedArray dst, int c, int[] indices, int cindices)
+        {
+            fixed (float* pDstStart = &dst.Items[0])
+            fixed (int* pidx = &indices[0])
+            {
+                float* pdst = Ptr(dst, pDstStart);
+
+                // REVIEW NEEDED: This line expands to (void)(c); but is it necessary?
```

Comment:
This is not necessary in C#.  It is used in C++ to get the compiler to stop complaining that `c` is only used in an Assert.

Reply:
Thanks - I will remove these two comments.  Same for below.